modules/EnsEMBL/Web/Component/Variation/Phenotype.pm

```diff
@@ -32,13 +32,16 @@ sub content {
   my $self = shift;
   my $object = $self->object;
   my $vf     = $self->hub->param('vf');
+  my $freq_data = $object->freqs;
+  my $pop_freq  = $self->format_frequencies($freq_data);
+
   my $html;
   ## first check we have uniquely determined variation
-  return $self->_info('A unique location can not be determined for this variant', $object->not_unique_location) if $object->not_unique_location;
+  return $self->_info('A unique location can not be determined for this Variation', $object->not_unique_location) if $object->not_unique_location;
 
   my $data = $object->get_external_data();
 
-  return 'We do not have any external data for this variant' unless (scalar @$data);
+  return 'We do not have any external data for this variation' unless (scalar @$data);
```

Comment:
Are you sure you want to change 'variant' back to 'variation'? (on line 40 too). This was changed by Fiona few hours ago.